playground/mocks/data/idp/idx/authenticator-enroll-ov-email.json

```diff
@@ -0,0 +1,173 @@
+{
+  "stateHandle": "02vWoN3BXxJ9oeJA05wdCO2sp1q-RimAkwgnN-6xaW",
+  "version": "1.0.0",
+  "expiresAt": "2020-04-09T16:15:46.000Z",
+  "step": "ENROLL",
+  "intent": "LOGIN",
+  "remediation": {
+    "type": "array",
+    "value": [
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "enroll-poll",
+        "href": "http://localhost:3000/idp/idx/challenge/poll",
+        "method": "POST",
+        "accepts": "application/vnd.okta.v1+json",
+        "refresh": 4000,
+        "value": [
+          {
+            "name": "stateHandle",
+            "required": true,
+            "value": "022afk9OLqrN2DipVIuIxC0wqzuxMaFIbpOf6pcBh8",
+            "visible": false,
+            "mutable": false
+          }
+        ]
+      },
+      {
+        "rel": [
+          "create-form"
+        ],
+        "name": "enroll-with-another-channel",
```

Comment:
The name seems too general but it ought to be specific like `enroll-ov-with-another-channel` ?

Reply:
if we name the form 'enroll-with-another-channel' we can reuse it for other authenticators and differentiate them by using authenticatorType